src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/TraceUtils.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Configuration;
```

Comment:
nit: missing empty line

```suggestion

using System.Configuration;
```